### ISSUE TITLE
To-Do List

### ISSUE THREAD

**Issue by C-Elliott44** [C-Elliott44/wordpress-flow#1]
### Get Info Needed
- [x] Ask User (in terminal) what the "Client's Domain" is
- [x] Ask User (in terminal) what the "WP Password" is
- [x] Ask User (in terminal) what the "WP Prefix" is

### Safety Check
- [x] Check if value for {Client's Domain} already exists in htdocs (aka my xxamp folder)
     - [x] If it already exists, ask User if they want to rename that folder already in htdocs
          - [x] YES, rename folder to {Client's Domain}-old
          - [x] NO, exit script and do nothing
          
### Download & Unzip WP Core
- [x] Download WP Core
- [x] Unzip WP Core
- [x] Rename unzipped WP Core folder to {Client's Domain}
- [x] Delete WP Core .zip after it has been Unzipped

### Create & Edit Config File
- [x] Create wp-config.php file using the existing wp-config-sample.php file
- [x] Update the default "sample values" in the newly created wp-config.php file
     - [x] Use {Client's Domain} for DB_NAME and DB_USER
          - [x] DB_NAME = {Client's Domain} remove extension (.com, .org, etc) - add _main
          - [x] DB_USER= {Client's Domain} remove extension (.com, .org, etc) - add _admin
     - [x] Use {WP Password} for DB_PASSWORD 
     - [x] Use {WP Prefix} for $table_prefix
     - [x] GET salts from https://api.wordpress.org/secret-key/1.1/salt/ and update sample salts

### Download & Stuff Theme From Underscores.me
- [x] download underscores theme from Github Repo: https://github.com/automattic/_s
- [x] Find and Replaces all the '_s' content
     - [x] Search for _s_ to capture all the functions names and replace with: megatherium_is_awesome_.
     - [x] Search for Text Domain: _s in style.css and replace with: Text Domain: megatherium-is-awesome.
     - [x] Search for  _s (with a space before it) to capture DocBlocks and replace with:  Megatherium_is_Awesome.
     - [x] Search for _s- to capture prefixed handles and replace with: megatherium-is-awesome-.
     - [x] Search for _S_ (in uppercase) to capture constants and replace with: MEGATHERIUM_IS_AWESOME_.

### Download all needed Files
Steps So Far
- Create "assets" folder
- Create folder called "bootstrap"
- Download Bootstrap
- Create

Should probably create a Dhali Repo to copy from instead of direct source
- [ ] Bootstrap
- [ ] Bootstrap Nav Walker
- [ ] Google Fonts (direct download)

### Create theme.json 
- [ ] probably make a template git directory to download then edit
     - [ ] specifically using this to create color pallets, font sizes, and other editor "save this" kind of stuff
     basically the Liia and Liz tool kit settings

### Import Custom Headers, Footers, and other "Stuff" That I Can Template Out
- [ ] Makes sample file of all "Template Stuff" for testing
 
### THINGS OF MORE LATER

DONE - But dont use until rest is complete
- [x] Move completed WP files to htdocs
